impeller/renderer/graphics_queue.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_IMPELLER_RENDERER_GRAPHICS_QUEUE_H_
+#define FLUTTER_IMPELLER_RENDERER_GRAPHICS_QUEUE_H_
+
+#include <functional>
+
+#include "fml/status.h"
+#include "impeller/renderer/command_buffer.h"
+
+namespace impeller {
+
+/// @brief An interface for submitting command buffers to the GPU for
+///        encoding and execution.
+class GraphicsQueue {
```

Comment:
From the name, I was expecting this to be akin to the graphics vs compute vs transfer queues. But this doesn't seem to be the case. How about just calling this `CommandQueue` instead?

Reply:
CommandQueue SGTM, solves naming problems too.

Reply:
Renamed, other stuff WIP